content/en/user-guide/chaos-engineering/fault-injection-service/index.md

```diff
@@ -1,50 +1,49 @@
 ---
-title: "Using the Fault Injection Service"
-linkTitle: "Using the Fault Injection Service"
-weight: 1
-description: Use LocalStack Outages Extension to mimic service outages by testing your infrastructure's ability to deploy robustly and recover from unexpected events.
+title: "Fault Injection Service"
```

Comment:
prefix with `AWS` for better differentiate from our chaos features

Reply:
Added in 3eba29459